ports/servoshell/app.rs

```diff
@@ -208,16 +216,36 @@ impl App {
             // Handle the event
             let mut consumed = false;
             if let Some(mut minibrowser) = app.minibrowser() {
-                if let winit::event::Event::WindowEvent { ref event, .. } = e {
-                    let response = minibrowser.context.on_event(&event);
-                    if response.repaint {
-                        // Request a winit redraw event, so we can recomposite, update and paint the
-                        // minibrowser, and present the new frame.
-                        window.winit_window().unwrap().request_redraw();
-                    }
-
-                    // TODO how do we handle the tab key? (see doc for consumed)
-                    consumed = response.consumed;
+                match e {
```

Comment:
```suggestion
                match event {
```

Reply:
done in 04a3def60440c7d9ec32d3f109ef8ab40da9f57c